orderer/common/cluster/clusterservice.go

```diff
@@ -167,26 +173,37 @@ func (s *ClusterService) handleMessage(stream ClusterStepStream, addr string, ex
 	if request == nil {
 		return errors.Errorf("Request message is nil")
 	}
+	if streamRefTime != atomic.LoadInt64(&s.MembershipByChannel[channel].LastUpdatedTime) {
+		return io.EOF
```

Comment:
I'm not sure io.EOF is the right error to return here. How about some error saying the stream is stale?